### PR TITLE
fix(app): add cursor and pageLength to GET /commandErrors

### PR DESCRIPTION
**Pull request by TamarZanzouri**
# Overview

closes [RQA-3031](https://opentrons.atlassian.net/browse/RQA-3031) and [RQA-3047](https://opentrons.atlassian.net/browse/RQA-3047).
pass in cursor and page length to get run command errors. 

## Test Plan and Hands on Testing

- upload a protocol that will enter ER.
- make sure there are a few errors to show.
- at the end of the run make sure when you click 'view error details' is showing the full error list.

## Changelog

added cursor and pageLength to GET request.

## Risk assessment

None.

[RQA-3031]: https://opentrons.atlassian.net/browse/RQA-3031?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[RQA-3047]: https://opentrons.atlassian.net/browse/RQA-3047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/organisms/Devices/ProtocolRun/ProtocolRunHeader.tsx [hunk 1]
```diff
@@ -116,6 +116,7 @@ import type {
   RunCommandErrors,
   RunError,
   RunStatus,
+  GetCommandsParams,
```
Comment (TamarZanzouri) [inline app/src/organisms/Devices/ProtocolRun/ProtocolRunHeader.tsx]:
```suggestion
```

File: app/src/organisms/Devices/ProtocolRun/ProtocolRunHeader.tsx [hunk 2]
```diff
@@ -677,9 +682,8 @@ function ActionButton(props: ActionButtonProps): JSX.Element {
   )
   const [showIsShakingModal, setShowIsShakingModal] = React.useState(false)
   const isSetupComplete =
-    isCalibrationComplete &&
-    isModuleCalibrationComplete &&
-    missingModuleIds.length === 0
+    //isCalibrationComplete &&
```
Comment (TamarZanzouri) [inline app/src/organisms/Devices/ProtocolRun/ProtocolRunHeader.tsx]:
```suggestion
    isCalibrationComplete &&
```

**Review by mjhuff — CHANGES_REQUESTED** [review]
I believe we need this on the ODD, too!

**Review by mjhuff — APPROVED** [review]
Noice

**Pull request merged**